client/py/synnax/simulation/auto.py

```diff
@@ -0,0 +1,143 @@
+import time
+from typing import Protocol
+import synnax as sy
+
+class SimulatedResponse(Protocol):
+    def __init__(self):
+        ...
+    def step(self, state: dict[str, float]) -> dict[str, float]:
+        ...
+
+class Anomaly(Protocol):
+    def __init__(self):
+        ...
+    def step(self, state: dict[str, float], errorStep: int) -> dict[str, float]:
```

Comment:
`errorStep` should be passed into init. Step must have the same signature. Variables should also use `snake_case`.